ovs/proto_trace_test.go

```diff
@@ -126,6 +126,86 @@ Datapath actions: drop`,
+    resubmit(,25)
+25. ip,vlan_tci=0x0000/0x1fff,nw_src=10.36.96.36, priority 2000, cookie 0x1e24001800000
+    push_vlan:0x8100
+    set_field:4118->vlan_vid
+    resubmit(,25)
+25. ip,dl_vlan=22,nw_dst=10.36.96.37, priority 2020, cookie 0x1d97c01800000
+    pop_vlan
+    resubmit(,28)
+28. ip,in_port=6, priority 110, cookie 0x1e24001900000
+    ct(table=30)
+    drop
+     -> A clone of the packet is forked to recirculate. The forked pipeline will be resumed at table 30.
+
+Final flow: ip,in_port=6,vlan_tci=0x0000,dl_src=56:03:b3:97:ac:c8,dl_dst=4a:72:d2:56:78:d1,nw_src=10.36.96.36,nw_dst=10.36.96.37,nw_proto=0,nw_tos=0,nw_ecn=0,nw_ttl=0
+Megaflow: recirc_id=0,eth,ip,in_port=6,vlan_tci=0x0000/0x1fff,dl_src=56:03:b3:97:ac:c8,dl_dst=00:00:00:00:00:00/01:00:00:00:00:00,nw_src=10.36.96.36,nw_dst=10.36.96.37,nw_proto=0,nw_frag=no
+Datapath actions: ct,recirc(0x2)
+
+===============================================================================
+recirc(0x2) - resume conntrack with ct_state=est|trk
+===============================================================================
+
+Flow: recirc_id=0x2,ct_state=est|trk,ct_mark=0x1e240,eth,ip,in_port=6,vlan_tci=0x0000,dl_src=56:03:b3:97:ac:c8,dl_dst=4a:72:d2:56:78:d1,nw_src=10.36.96.36,nw_dst=10.36.96.37,nw_proto=0,nw_tos=0,nw_ecn=0,nw_ttl=0
+
+bridge("br0")
+-------------
+    thaw
+        Resuming from table 30
+30. ct_state=+est+trk,ct_mark=0x1e240,in_port=6, priority 220, cookie 0x1e24001900000
+    resubmit(,35)
+35. priority 100
+    resubmit(,45)
+45. priority 100
+    resubmit(,50)
+50. priority 100
+    resubmit(,60)
+60. ip,in_port=6,dl_src=56:03:b3:97:ac:c8,nw_src=10.36.96.36, priority 1020, cookie 0x1e24001800000
+    resubmit(,62)
+62. ip,tun_id=0,nw_dst=10.36.96.0/20, priority 1000
+    resubmit(,67)
+67. conj_id=2, priority 1500, cookie 0x200020000
+    set_field:0x2->tun_id
+    resubmit(,68)
+68. ip,tun_id=0x2,nw_dst=10.36.96.37, priority 1500, cookie 0x200020000
+    set_field:10.39.129.11->tun_dst
+    output:9
+     -> output to native tunnel
+     >> native tunnel routing failed
+
+Final flow: recirc_id=0x2,ct_state=est|trk,ct_mark=0x1e240,eth,ip,tun_src=0.0.0.0,tun_dst=10.39.129.11,tun_ipv6_src=::,tun_ipv6_dst=::,tun_gbp_id=0,tun_gbp_flags=0,tun_tos=0,tun_ttl=0,tun_flags=0,in_port=6,vlan_tci=0x0000,dl_src=56:03:b3:97:ac:c8,dl_dst=4a:72:d2:56:78:d1,nw_src=10.36.96.36,nw_dst=10.36.96.37,nw_proto=0,nw_tos=0,nw_ecn=0,nw_ttl=0
+Megaflow: recirc_id=0x2,ct_state=+est+trk,ct_mark=0x1e240,eth,ip,tun_id=0,tun_dst=0.0.0.0,in_port=6,dl_src=56:03:b3:97:ac:c8,dl_dst=4a:72:d2:56:78:d1,nw_src=10.36.96.36,nw_dst=10.36.96.37,nw_ecn=0,nw_frag=no
+Datapath actions: drop`,
+			datapathActions: NewDataPathActions("drop"),
+			flowActions: []string{
+				"resubmit(,25)",
+				"push_vlan:0x8100",
+				"set_field:4118->vlan_vid",
+				"resubmit(,25)",
+				"pop_vlan",
+				"resubmit(,28)",
+				"ct(table=30)",
+				"drop",
```

Comment:
Will this `drop` action still being included cause problems for things consuming these `flowActions`?

Reply:
That's a good question, the way I see it is that the purpose of this function is to simply split the "actions" and returns the list. Something then consuming this list, like we do in the datapath validation framework can build state and bypass "drop" when it comes after a "ct()" flow action. What do you think ?

Reply:
Yea I'm still a little hazy on what will break the datapath validation. Just wanted to make sure you saw and weren't scared by it.

Reply:
I was thinking our framework should probably assert that ct is always followed by drop. At least, that is the way I've seen it. What do you think? But yeah, this code this function should just split the actions.

Reply:
nm, I found an example where ct was following by a set_field to set the ct_mark.